packages/faustwp-core/src/getWordPressProps.tsx

```diff
@@ -141,6 +144,25 @@ export async function getWordPressProps(
     });
   }
 
+  let queries: { [key: string]: string } | null = null;
+  if (template.queries) {
+    const queryCalls = template.queries.map((query) => {
+      return client.query({
+        query,
+        variables: templateVariables,
```

Comment:
This is the tricky part. Each query may need different variables and not we are bundling all the potential variables on each requests. This could get out of hand soon.